appdatabase/migrations/sql/1676368933_keypairs_to_keycards.up.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE IF NOT EXISTS keycards (
+    keycard_uid VARCHAR NOT NULL PRIMARY KEY,
+    keycard_name VARCHAR NOT NULL,
+    keycard_locked BOOLEAN DEFAULT FALSE,
+    key_uid VARCHAR NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS keycards_accounts (
+    keycard_uid VARCHAR NOT NULL,
+    account_address VARCHAR NOT NULL,
+    FOREIGN KEY(keycard_uid) REFERENCES keycards(keycard_uid) 
+      ON UPDATE CASCADE 
+      ON DELETE CASCADE    
+);
+
+INSERT INTO keycards select distinct keycard_uid, keycard_name, keycard_locked, key_uid from keypairs;
+
+INSERT INTO keycards_accounts select keycard_uid, account_address from keypairs;
+
+DROP TABLE keypairs;
```

Comment:
does this need a sync with mobile team?

Reply:
I think no, they are not using this yet. Anyhow I've already posted a link to this PR in #status-go channel. 
@qfrank @rasom could you review/confirm?

Reply:
I'm not familiar with this, maybe @rasom knows better, but if u mean if mobile used api `GetMigratedKeyPairByKeyUID`, i'd say no since i just checked.